apps/evans/pages/_app.tsx

```diff
@@ -1,19 +1,13 @@
-import Script from 'next/script';
 import type { AppProps } from 'next/app';
 import '@/styles/globals.css';
 import '@/styles/cover.css';
+import '@/styles/fontawesome.css';
 import Chrome from '@/components/chrome/Chrome';
 
 const App: React.FC<AppProps> = ({ Component, pageProps }) => (
-  <>
-    <Script
-      src="https://kit.fontawesome.com/597740db7b.js"
-      strategy="beforeInteractive"
-    ></Script>
```

Comment:
removing the full font-awesome integration saves about 50kb of unused CSS on every page load. this optimization had been made in our gatsby site, this PR just restores it.